svc-events/events/evtsubscription.go

```diff
@@ -542,7 +542,8 @@ func (p *PluginContact) eventSubscription(postRequest evmodel.RequestBody, origi
 		EventHostIP:    deviceIPAddress,
 		OriginResource: origin,
 	}
-	if !(strings.Contains(locationHdr, target.ManagerAddress)) {
+	hosts := strings.Split(target.ManagerAddress, ":")
```

Comment:
Wouldn't it be better to use net.SplitHostPort to maintain consistency and also a better option than strings.Split for hostport type?
Also, shouldn't location header carry the valid address? If port is removed, the location can't be reached.

Reply:
added changes using net.SplitHostPort 
and, 
since we are storing event subscription details in DB against IP or FQDN itself, hence removing Port from eventSubscription.location